learningequality/studio
# File fixes

## Description

- Fixed problems with uploading subtitles
- Fixed removing files
- Fixed validation to only happen when user clicks save

## Steps to Test
Subtitle Uploading
1. Upload a video
2. Upload vtt subtitles
3. Save and Close
4. Check if files are still there

Removing Files
1. Add two videos- a high and low resolution
2. Save
3. Reopen and remove one of the videos
4. Save and Close
5. Reopen and see if only one video is there

## Checklist

- [ ] Have the changes been added to the [CHANGELOG](https://github.com/learningequality/studio/blob/master/CHANGELOG.md)?
